.gitignore

```diff
@@ -81,3 +81,6 @@ celerybeat-schedule.*
 geonode\.tests\.bdd\.e2e\.test_login/
 
 *.log
+
+/celerybeat.pid
+/celerybeat-schedule
```

Comment:
these changes are not related are they ?

Reply:
Yep, sorry. Will remove it.